frontends/main/next.config.js

```diff
@@ -79,4 +81,41 @@ const nextConfig = {
   },
 }
 
-module.exports = nextConfig
+// Injected content via Sentry wizard below
+
+const { withSentryConfig } = require("@sentry/nextjs")
+
+module.exports = withSentryConfig(nextConfig, {
```

Comment:
This, and a lot of stuff below, was injected automatically by `@sentry/wizard` CLI